pkgs/applications/networking/cluster/k3sup/default.nix

```
@@ -0,0 +1,58 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, makeWrapper
+, bash
+, openssh
+}:
+
+buildGoModule rec {
+  pname = "k3sup";
+  version = "0.11.3";
+
+  src = fetchFromGitHub {
+    owner = "alexellis";
+    repo = "k3sup";
+    rev = version;
+    sha256 = "sha256-6WYUmC2uVHFGLsfkA2EUOWmmo1dSKJzI4MEdRnlLgYY=";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  overrideModAttrs = (_: {
+    postBuild = ''
+      substituteInPlace vendor/github.com/alexellis/go-execute/pkg/v1/exec.go \
+        --replace "/bin/bash" "bash"
```

```suggestion
        --replace "/bin/bash" "${bash}/bin/bash"
```

And it can be removed from `makeBinPath`.